plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/JavaUtils.java

```diff
@@ -92,28 +139,6 @@ public static void checkPackageName(String name) throws IllegalStateException {
      * @return <code>true</code> if the specified text is a valid package name, <code>false</code> otherwise
      */
     public static boolean isValidPackageName(String name) {
-        IStatus status = validatePackageName(name);
-        switch (status.getSeverity()) {
-            case Status.WARNING:
-            case Status.OK:
-                return true;
-            default:
-                return false;
-        }
-    }
-
-    private static IStatus validateCompilationUnitName(String name) {
-//        return JavaConventions.validateCompilationUnitName(name, JavaCore.getOption(JavaCore.COMPILER_SOURCE),
```

Comment:
The porting JavaConventions to client complicates client side validation code. Based on this comments     
https://github.com/eclipse/che/pull/513/files#diff-092710f973f752e67aa777e2cec6c3bcL115

we've already had JavaConventions validation on client side, but it was complicated.
In my implementation i use regex to validate packages and class names in according to java name convention.